lib/page-object/platforms/selenium_webdriver/page_object.rb

```diff
@@ -1071,6 +1087,40 @@ def find_selenium_elements(identifier, type, tag, other=nil)
           elements.map { |element| type.new(element, :platform => :selenium_webdriver) }
         end
 
+        def find_selenium_pages(identifier, page_class)
```

Comment:
A lot of this is lifted and adapted from the code for finding an element or a collection of elements. This is only a small departure in behavior from that, so there's a lot of overlap. Ideally the duplication could be extracted as a private method, but I wasn't sure if you would want me to go that far.